src/Kdyby/Doctrine/Proxy/ProxyAutoloaderListener.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Doctrine\Proxy;
+
+use Kdyby;
+use Nette;
+
+/**
+ * @author Jáchym Toušek <enumag@gmail.com>
+ */
+class ProxyAutoloaderListener extends Nette\Object implements Kdyby\Events\Subscriber
+{
+
+	/** @var array */
+	private $proxyAutoLoaders;
+
+
+
+	public function __construct(array $proxyAutoLoaders)
+	{
+		$this->proxyAutoLoaders = $proxyAutoLoaders;
```

Comment:
I don't really have a preference, but in the original code it was "ProxyAutoloader"